JS/Lesson25/goksel/filter.js

```diff
@@ -332,6 +358,10 @@ const studentsExercise9 = [
 	},
 ];
 
+const gradeFiltered = studentsExercise9.filter((student) => student.grade == "D"|| student.grade == "C" || student.grade == "D+");
```

Comment:
There's a shorter solution:
```suggestion
const gradeFiltered = studentsExercise9.filter((student) => student.grade[0].toUpperCase() !== "D");
```
Get the first letter of any grade and convert to upper case, then compare it to D. You could also add a check for letters E and F as they are possible grades.

Reply:
@Gokselagan this comment is not resolved yet, please take another look